common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -174,9 +174,12 @@ object TransactionId {
  * A thread-safe transaction counter.
  */
 trait TransactionCounter {
+    val numberOfInstances: Int
+    val instance: Int
+
     def transid(): TransactionId = {
-        TransactionId(cnt.incrementAndGet())
+        TransactionId(cnt.addAndGet(numberOfInstances))
     }
 
-    private val cnt = new AtomicInteger(1)
+    private val cnt = new AtomicInteger(numberOfInstances + instance)
```

Comment:
This is just setting the initial value however isn't it? Don't you mean to use this as a delta on the increment; or am I missing something?